Objects/exceptions.c

```diff
@@ -2748,40 +2748,78 @@ unicode_error_adjust_end(Py_ssize_t end, Py_ssize_t objlen)
     return end;
 }
 
+#define _PyUnicodeError_CAST(PTR)   _Py_CAST(PyUnicodeErrorObject *, PTR)
```

Comment:
I'm ok with using the macro, but the [precedent](https://github.com/search?q=repo%3Apython%2Fcpython%20_Py_CAST&type=code) is to not use `_Py_CAST` in source files (with the exception of a few weird objects). Really, I'm not totally sure why we have this macro in the first place, because it doesn't change at all depending on compilers (it's always just a C style cast). Why not just make this `(PyUnicodeErrorObject *)(PTR)`?

Reply:
Oh yes. I just C/C actually but _Py_CAST is not needed at all.